README.md

```diff
@@ -27,6 +27,10 @@ This stylesheet is transforming an XML document by removing any "sec" element wi
 
 TODO: We need an expression of whether we believe this is work for biorXiv, encoda or EPP team.
 
+### [/src/2022.05.30.22275761/add-missing-aff-for-AK-v1.xsl](/src/2022.05.30.22275761/add-missing-aff-for-AK-v1.xsl)
+
+This xsl is adding a missing affiliation for the first author. Affiliations a linked to using an `<xref>` element, which is a child of the author's `<contrib contrib-type="author">` element. This was presumably a typesetting error.
```

Comment:
Is this something we could reasonably expect biorXiv to address at some point or is it a weakness in the submission to biorXiv?

Reply:
It's an error from bioRxiv's typesetters. I think we need to establish how we'd communicate this to bioRxiv, but in theory they should be able to simply fix this in the future.

This one specifically was one of the 'examples' we launched with back in October, and has now been published as an (old style) VOR, so I'm not sure how we want to specifically handle it.

Reply:
> It's an error from bioRxiv's typesetters. I think we need to establish how we'd communicate this to bioRxiv, but in theory they should be able to simply fix this in the future.

Simply adding this comment would improve the documentation. We will be seeking to direct biorXiv's attention to all these adjustments that we have had to make.

Reply:
OK I have update the readme